\documentclass[_main.tex]{subfiles}
 
\begin{document}

\noindent\fbox{%
    \parbox{\textwidth}{%

\textbf{This glossary explains the terminology used in this paper.}  Note that this sometimes differs from common usage.

%\begin{small}

\medskip

\textbf{Allele.}  An instance of the parasite genome, e.g. $n$ haploid individuals correspond to $n$ alleles.

\medskip

\textbf{Coalescence.}  If two lineages are traced back in time, coalescence occurs when they meet in the same ancestral allele.

\medskip

\textbf{Cotransmission.}  Occurs when a host or a vector is carrying a mixture of parasite alleles with different ancestral histories, and this is not directly due to superinfection.

\medskip

\textbf{Crossing rate of transmission chains ($\chi$).}  The proportion of hosts that acquire parasites from more than one transmission chain, i.e. from more than one host in the previous generation.  This is equivalent to proportion of hosts that are superinfected.

\medskip

\textbf{Effective number of hosts ($N_h$).}   The number of hosts that effectively transmit parasites in each generation of the transmission graph.  This is a form of population bottleneck. 

\medskip

\textbf{Effective recombination.}  Recombination between genetically distinct alleles that acts to change the DNA sequence of a haplotype locus.

\medskip

\textbf{Effective recombination parameter $\phi_t$.}  The probability that, if recombination occurs at a locus at time $t$, this will change the DNA sequence of the locus.

\medskip

\textbf{Haplotype.}  A specific DNA sequence observed at a haplotype locus.

\medskip

\textbf{Haplotype locus.}  A locus that extends over multiple nucleotide positions.

\medskip

\textbf{Heterozygosity ($H$).}  The probability that two alleles sampled randomly from some population are heterozygous, i.e. that they have different DNA sequences. 

\medskip

\textbf{Homozygosity ($G$).}  The probability that two alleles sampled randomly from some population are homozygous, i.e. that they have the same DNA sequence.

\medskip

\textbf{Lineage.} A path that traces the ancestry of an allele at a point locus, going backwards in time through the transmission graph.  A point locus is not affected by recombination, so a lineage can be traced back over many generations despite frequent recombination events.

\medskip

\textbf{Locus.}  A specific location in the genome.  This can be either a single nucleotide position (a point locus) or a sequence extending over multiple nucleotide positions (a haplotype locus).

\medskip

\textbf{Nucleotide diversity ($\pi$).}  The probability that two alleles are heterozygous at a random nucleotide position in the genome.

\medskip

\textbf{Point locus.}  A specific single nucleotide position in the genome. 

\medskip

\textbf{Quantum of transmission ($Q$).} The number of parasite alleles transmitted from one host to the next via a vector.  $Q$ summarises a complex series of bottlenecks in host-vector and vector-host transmission occurring during one generation of the parasite life-cycle.

\medskip

\textbf{Superinfection.}  Infection of a host with parasites from more than one source in the previous generation.  In the genomic transmission graph this is equivalent to crossing of transmission chains.

\medskip

\textbf{Transmission bottleneck.}  A population bottleneck that affects the number of alleles that are passed from one generation of the genomic transmission graph to the next.  The quantum of transmission $Q$ and the effective number of hosts $N_h$ can both be considered as transmission bottlenecks.

\medskip

\textbf{Transmission chain.}  A sequence of host-to-host transmission events.   If we pick any node in the transmission graph, and trace a path forwards in time along the edges to another node, that is a transmission chain.

%\end{small}

    }%
}

\end{document}